### PR TITLE
Add note about 23.8 sumo release unavailability

### PR DESCRIPTION
**Pull request by mike-petersen-ni**
See [AB#2561580](https://dev.azure.com/ni/94b22d7b-ad7b-4f5e-88f0-867910f91c94/_workitems/edit/2561580)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -279,6 +279,8 @@ Branch: `nilrt/22.5/hardknott`
 ## 8.17
 Branch: `nilrt/23.8/sumo`
 
+**Note**: this version was not propagated to the 23.8 system image installers. Running the 23.8 installer will install the 8.16/23.5 ARM base system image.
+
 ### meta-nilrt
 #### Changed
 - Socketcan interfaces will [no longer](https://github.com/ni/meta-nilrt/pull/587) be started with ifplugd.
```
Comment (chaitu236) [inline CHANGELOG.md]: This block is a duplicate of one below. Was this in error @amstewart?

Comment (mike-petersen-ni) [reply CHANGELOG.md]: I also noticed this; 794a58bf162f seems to be a cherrypick of an existing commit.

Comment (amstewart) [reply CHANGELOG.md]: Oh whoops. Could you just delete one of them?

Comment (mike-petersen-ni) [reply CHANGELOG.md]: Done in v2

**Comment by mike-petersen-ni** [comment]
## v2
 - Reverted commit that duplicated the sumo 23.8 section

**Review by amstewart — APPROVED** [review]

**Review by chaitu236 — APPROVED** [review]

**Pull request merged**